bin/doc-gen.py

```diff
@@ -684,7 +684,13 @@ def parse_data_models_from_search(search):
     gmd = args.gen_markdown_docs
 
     stories = load_objects("stories/*.yml")
-    detections = load_objects("detections/*.yml")
+    detections = []
+    application = load_objects("detections/application/*.yml")
```

Comment:
It may be easy to miss one of these files if we add a new category. Could also just `load_objects("detections/*/*.yml")` since we pass that to `glob.glob`.

Reply:
agree, much cleaner way to do this, fixed in all bin scripts I touched. Thank you!